pyphare/pyphare/core/box.py

```diff
@@ -104,6 +104,10 @@ def refine(box, ratio):
     return Box(box.lower * ratio, box.upper * ratio + ratio - 1)
 
 
+def coarsen(box, ratio):
+    return Box(box.lower / ratio, ((box.upper + 1) / ratio) - 1)
```

Comment:
is this right?

Reply:
(20,41) becomes (20/2, 42/2-1) = (10,20), looks ok

Reply:
should assert something like?

```
assert (box.lower % ratio == 0).all() and assert (box.upper % ratio == 0).all())
```

possible overkill but just in case maybe

Reply:
well I think we will always want to coarsen boxes that describe "refined index space", i.e. that are associated with levels > L0. Since our refinement ratio is 2 we will always have the lower index that can be divided by 2. However the upper is next_coarser.upper*2+1 so not a multiple of 2 (ratio)